src/utils/point_locator_tree.C

```diff
@@ -225,7 +229,7 @@ const Elem * PointLocatorTree::operator() (const Point & p,
               this->_element =
                 this->perform_linear_search(p,
                                             allowed_subdomains,
-                                            /*use_close_to_point*/ true,
```

Comment:
Is this comment actually wrong? If so, please fix it, if not, please don't delete it because it lets us know what the parameter is used for at a glance...

Reply:
Sorry, I interpreted that as the actual variable _use_close_to_point_tol rather than a comment and thought I was doing a little cleanup.